sdk/monitor/monitor-opentelemetry/README.md

```diff
@@ -0,0 +1,5 @@
+## Contributing
```

Comment:
Please use the README-TEMPLATE from the template project as a starter

Reply:
Updated README with initial iteration